docs/source/rules.rst

```diff
@@ -133,6 +133,14 @@ req_subkeys
 
 This feature should be avoided, but it is useful if you defined a loose schema to trade flexibility for safety; see `Schemas <conf-schemas.html#json-example-osquery>`_.
 
+context
+~~~~~~~~~~~
+
+``context`` is an optional argument which defines an extra field of information to pass on inside
```

Comment:
Can probably be simplified to something like: `context is an optional field to pass extra instructions to the alert processor on how to route the alert.`